tests/CascadeSoftDeletesIntegrationTest.php

```diff
@@ -68,6 +83,25 @@ public function it_cascades_deletes_when_deleting_a_parent_model()
         $this->assertCount(0, Tests\Entities\Comment::where('post_id', $post->id)->get());
     }
 
+    /* Many2Many with pivot table(deletes entries in pivot) */
+    public function it_cascades_deletes_entries_from_pivot_table()
+    {
+        $author = Tests\Entities\Author::create(['name' => 'ManyToManyTestAuthor']);
+
+        $this->attachPostTypesToAuthor($author);
+        $this->assertCount(2, $author->posttypes);
+
+        $author->delete();
+
+        $manager = Manager::$instance;
```

Comment:
`$instance` is a protected property and can't be accessed like this. I'm not sure what you're trying to achieve here.